stripe/src/main/java/com/stripe/android/model/AccountParams.java

```diff
@@ -0,0 +1,79 @@
+package com.stripe.android.model;
+
+import android.support.annotation.NonNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.stripe.android.StripeNetworkUtils.removeNullAndEmptyParams;
+
+/**
+ * Represents a grouping of parameters needed to create a Token for a Connect account on the server.
+ */
+public class AccountParams {
+
+    static final String API_PARAM_LEGAL_ENTITY = "legal_entity";
+    static final String API_TOS_SHOWN_AND_ACCEPTED = "tos_shown_and_accepted";
+    private Boolean mTosShownAndAccepted;
+    private Map<String, Object> mLegalEntity;
+
+    /**
+     * @param tosShownAndAccepted indicates that the platform showed the user the appropriate text
+     *                            and links to Stripe's terms of service. Tokens will only generated
+     *                            when this is true.
+     * @param legalEntity map that specifies the legal entity for which the connect account is being
+     *                    created. Can contain any of the field specified by legal_entity is the API
+     *                    docs.
+     *                    See {@linktourl https://stripe.com/docs/api#account_object-legal_entity}
+     * @return
+     */
+    public static AccountParams createAccountParams(
+            boolean tosShownAndAccepted,
+            Map<String, Object> legalEntity) {
+        AccountParams accountParams = new AccountParams()
+                .setTosShownAndAccepted(tosShownAndAccepted)
+                .setLegalEntity(legalEntity);
+        return accountParams;
+    }
+
+    /**
+     * @param tosShownAndAccepted whether the platform showed the user the appropriate text
+     *                            and links to Stripe's terms of service. Tokens will only generated
+     *                            when this is true.
+     * @return {@code this}, for chaining purposes
+     */
+    public AccountParams setTosShownAndAccepted(boolean tosShownAndAccepted) {
+        mTosShownAndAccepted = tosShownAndAccepted;
+        return this;
+    }
+
+    /**
+     * @param legalEntity map that specifies the legal entity for which the connect account is being
+     *                    created. Can contain any of the field specified by legal_entity is the API
+     *                    docs.
+     *                    See {@linktourl https://stripe.com/docs/api#account_object-legal_entity}
+     * @return {@code this}, for chaining purposes
+     */
+    public AccountParams setLegalEntity(Map<String, Object> legalEntity) {
+        mLegalEntity = legalEntity;
```

Comment:
Any concerns about validating that the user hasn't stuck something non-serializable into the legalEntity?

For example, I'm not sure what happens if they try to use `com.stripe.android.model.Address` objects instead of a `Map<String, Object>` for `legal_entity.address`.

I followed some of the code down, and it *looked* like the networking code was going to eventually call to `flattenParamsValue()`, which I don't think handles model objects.